assets/js/linear_algebra/dot_product_formula.js

```diff
@@ -128,39 +129,39 @@ function plot(scatter, axis, tt){
           [[{text: u.coord.x.toFixed(2), key: 'xu'}],
            [{text: u.coord.y.toFixed(2), key: 'yu'}],
            [{text: u.coord.z.toFixed(2), key: 'zu'}]], 
-          [start_coord_x, start_coord_y], 14, 5
+          [start_coord_x, start_coord_y], 14
           ),
       [lines_plus, texts_plus] = lib.text_matrix_to_list(
           [[{text: '\uFE62', text_color: 0, key: 'fplus'}],
            [{text: '\uFE62', text_color: 0, key: 'splus'}]], 
-          [start_coord_x - 0.3 * w_unit, start_coord_y], 14, 5
+          [start_coord_x - 0.3 * w_unit, start_coord_y], 14
           ),
       [lines_multi, texts_multi] = lib.text_matrix_to_list(
           [[{text: '\u00D7', text_color: 0, key: 'fmulti'}],
            [{text: '\u00D7', text_color: 0, key: 'smulti'}],
            [{text: '\u00D7', text_color: 0, key: 'tmulti'}]], 
-          [start_coord_x + 0.6 * w_unit, start_coord_y], 14, 5
+          [start_coord_x + 0.6 * w_unit, start_coord_y], 14
           ),
       [lines_v, texts_v] =  lib.text_matrix_to_list(
           [[{text: v.coord.x.toFixed(2), key: 'xv'}],
            [{text: v.coord.y.toFixed(2), key: 'yv'}],
            [{text: v.coord.z.toFixed(2), key: 'zv'}]], 
-          [start_coord_x + 0.9 * w_unit, start_coord_y], 14, 15
+          [start_coord_x + 0.9 * w_unit, start_coord_y], 14
           );
 
   let z_texts = [{text: u.coord.z.toFixed(2), x: start_coord_x,
                   y: start_coord_y + 0.3 * h_unit, key: 'zu', text_opacity: 0},
                  {text: v.coord.z.toFixed(2), x: start_coord_x + 0.9 * w_unit,
                   y: start_coord_y + 0.3 * h_unit, key: 'zv', text_opacity: 0}],
       big_line = [{x: start_coord_x - 0.1 * w_unit,
-                   y: start_coord_y + 0.65 * h_unit, z: 0},
-                  {x: start_coord_x + 1.55 * w_unit,
-                   y: start_coord_y + 0.65 * h_unit, z: 0,
+                   y: end_coord_y + 0.12 * h_unit, z: 0},
+                  {x: start_coord_x + 1.6 * w_unit,
+                   y: end_coord_y + 0.12 * h_unit, z: 0,
                    color: 0}],
       uTv_texts = [{text: 'u\u1d40v =', x: start_coord_x - 0.6 * w_unit,
-                    y: start_coord_y + 0.9 * h_unit, text_color: 0, key: 'uTv_text'},
+                    y: end_coord_y + 0.37 * h_unit, text_color: 0, key: 'uTv_text'},
                    {text: uTv.toFixed(3), x: start_coord_x + 0.4 * w_unit,
-                    y: start_coord_y + 0.9, text_color: 0, key: 'uTv_numb'},];
+                    y: end_coord_y + 0.37 * h_unit, text_color: 0, key: 'uTv_numb'},];
```

Comment:
name for some of the constants will help with reading the code.

Reply:
done